### ISSUE TITLE
Accessibility Testing for MHV on VA, Medications, In Product Education (IPE)

### ISSUE THREAD

**Issue by BobbyBaileyRB** [department-of-veterans-affairs/va.gov-team#104933]
### Product information

- [x] Team name, product name, and feature name have been added to the title of this issue.
- [x] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

Bobby Bailey

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### Who completed the axe scans?

Bobby Bailey

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

![Image](https://github.com/user-attachments/assets/71c7cc19-9581-48df-8c64-104a4cb4435f)

### Who completed the content zoom and reflow test?

Bobby Bailey

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### Who completed the keyboard navigation test?

Bobby Bailey

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [ ] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

IPE does not follow logical tab order (bug sent to developers on 3/12/25).

### Do you have any other results to share?

**Finding: (also sent to the RX team and developers)**

1. Could be a blocker: There is no aria-describedby attribute linking the IPE text to its corresponding component. The first sentence of the component should be announced by screen readers when focusing on the related element, when it does, it must contain the most important content. This means that when someone focuses on the filter, the required text within the IPE should also be read aloud. Otherwise, people will not know there is a IPE since it lives under components.
